pandera/api/base/checks.py

```diff
@@ -83,9 +83,9 @@ def __setstate__(self, state):
 class MetaCheck(type):  # pragma: no cover
     """Check metaclass."""
 
-    BACKEND_REGISTRY: Dict[
-        Tuple[Type, Type], Type[BaseCheckBackend]
-    ] = {}  # noqa
+    BACKEND_REGISTRY: Dict[Tuple[Type, Type], Type[BaseCheckBackend]] = (
```

Comment:
is this due to updating black? would be preferable if this happened as a separate PR